013_IP_L141/013_IP_L141_unsafe.py

```
@@ -0,0 +1,305 @@
+    if request.method == 'POST':  
+        username = request.form['username']  
+        password = request.form['password'] 
+        with open(r'./login.txt','r') as f:
+            a=f.read()
+            a=a.split('\n')
+            for i in a:
+                usps=i.split(' ')
+                if usps[0]==username and usps[1]==password:
+                    resp = make_response(redirect('http://116.62.60.158/index', code=302))
+                    resp.set_cookie("login",username)
+                    #此处没有使用高级方法，以后修复
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)